### PR TITLE
Add some more misspellings found in Emacs

### PR DESCRIPTION
**Pull request by skangas**
Some more misspellings found in Emacs using an external spell checker.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -27876,6 +27891,7 @@ queus->queues
 quew->queue
 quickier->quicker
 quicklyu->quickly
+quicky->quickie
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:27894]: Perhaps `quickly` too?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -31886,6 +31902,7 @@ simliar->similar
 simliarly->similarly
 simlicity->simplicity
 simlified->simplified
+simlify->simplify
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:31905]: `simlifies`, `simlifying`?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:27894]: Added.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:31905]: Added.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**